sdk/ai/Azure.AI.Inference/tests/InferenceClientTelemetryTest.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.AI.Inference.Telemetry;
+using Azure.AI.Inference.Tests.Utilities;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Azure.AI.Inference.Tests
+{
+    public class InferenceClientTelemetryTest : RecordedTestBase<InferenceClientTestEnvironment>
+    {
+        private ChatCompletionsOptions m_requestOptions, m_requestStreamingOptions;
+
+        public InferenceClientTelemetryTest(bool isAsync) : base(isAsync)
+        { }
+
+        [SetUp]
+        public void Setup()
+        {
+            m_requestOptions = new ChatCompletionsOptions()
+            {
+                Messages =
+                {
+                    new ChatRequestSystemMessage("You are a helpful assistant."),
+                    new ChatRequestUserMessage("What is the capital of France?"),
+                },
+                Model = "gpt-4o",
+                Temperature = 1,
+                MaxTokens = 10
+            };
+            m_requestStreamingOptions = new ChatCompletionsOptions()
+            {
+                Messages =
+                {
+                    new ChatRequestSystemMessage("You are a helpful assistant."),
+                    new ChatRequestUserMessage(
+                        "Give me 5 good reasons why I should exercise every day."),
+                },
+                Model = "gpt-4o"
+            };
+            Environment.SetEnvironmentVariable(OpenTelemetryConstants.EnvironmentVariableSwitchName, "1");
+        }
+
+        public enum TestType
+        {
+            Plane,
+            Streaming
+        };
+
+        private ChatCompletionsClient CreateClient(Uri endpoint)
```

Comment:
Can these two private methods go lower down in the file, in a "helpers" section, like the other test file? To make it easier to distinguish between tests and helper functionality

Reply:
Fixed here and in other tests.